docs/reference/commandline/secret_create.md

```diff
@@ -16,23 +16,36 @@ keywords: ["secret, create"]
 # secret create
 
 ```Markdown
-Usage:  docker secret create [OPTIONS] SECRET
+Usage:	docker secret create [OPTIONS] SECRET
+
+Create a secret from a file or STDIN as content
 
-Create a secret using stdin as content
 Options:
-      --help         Print usage
-  -l, --label list   Secret labels (default [])
+  -f, --file string   Read from a file or STDIN ('-')
+      --help          Print usage
+  -l, --label list    Secret labels (default [])
 ```
 
-Creates a secret using standard input for the secret content. You must run this
+Creates a secret using standard input or from a file for the secret content. You must run this
 command on a manager node.
 
 ## Examples
 
 ### Create a secret
 
 ```bash
-$ cat secret.json | docker secret create secret.json
+$ cat secret.json | docker secret create -f - secret.json
```

Comment:
Why would you do it this way?? Just `docker secret create -f secret.json secret.json

Also these examples are made more confusing when you call the secret the same as the file name used to create it. So maybe: `docker secret create -f secret.json my_secret`